unit_test/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
I think removing this for now, merging the rest, then reformatting to go with #42 will be best

Reply:
I have removed this file.  The new unit_test/CMakeLists.txt looks overly complicated though - maybe it could use a few example lines for tests to explain the use options for CabanaMD_add_tests macro.